docs/3x/get_started.md

```diff
@@ -0,0 +1,89 @@
+# Getting Started
+
+1. [Quick Samples](#quick-samples)
+
+2. [Feature Matrix](#feature-matrix)
+
+## Quick Samples
+
+```shell
+# Install Intel Neural Compressor
```

Comment:
Please add this example to the regular test.

Reply:
Ok, I will.